recipes/gsl-lite/all/conanfile.py

```diff
@@ -40,34 +40,42 @@ def _contract_map(self):
             "unenforced": "GSL_UNENFORCED_ON_CONTRACT_VIOLATION"
         }
 
+    def config_options(self):
+        if Version(self.version) >= "1.0":
+            del self.options.on_contract_violation
+
     def layout(self):
-        basic_layout(self, src_folder="src")
+        cmake_layout(self, src_folder="src")
 
     def package_id(self):
         self.info.clear()
 
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
 
+    def generate(self):
+        tc = CMakeToolchain(self)
```

Comment:
Checking if this generates the same files our copy did

Reply:
Not quite sure what the question is, sorry.

The old recipe just copied some files from the source directory. The updated recipe now just uses the CMake install process, which I think makes more sense.

Reply:
Sorry, not a question but a note for future reviewers that this needs checking, sorry about the noise :)